docs/src/main/sphinx/connector/hive.rst

```diff
@@ -1101,6 +1101,11 @@ to the connector using a :doc:`WITH </sql/create-table-as>` clause::
       ``s3a://test/name=${name}/``. Mapped from the AWS Athena table property
       `storage.location.template <https://docs.aws.amazon.com/athena/latest/ug/partition-projection-setting-up.html#partition-projection-specifying-custom-s3-storage-locations>`_
     - ``${table_location}/${partition_name}``
+    -
+  * - ``extra_properties``
```

Comment:
As discussed offline, please add a paragraph in the connector docs about how the user can specify extra properties for a hive table  and also how they can be retrieved via "$properties" metadata table.
Follow-up PR though

cc @mosabua